nrf802154_sniffer/nrf802154_sniffer.py

```diff
@@ -92,9 +94,22 @@ def stop_sig_handler(self, *args, **kwargs):
             self.serial_queue.put(b'')
             self.serial_queue.put(b'sleep')
             self.running.clear()
+            
+            for thread in self.threads:
+                thread.join()
+
+            self.threads = []
         else:
             self.logger.warning("Asked to stop {} while it was already stopped".format(self))
 
+    @staticmethod
+    def get_hex_path():
+        """Returns path to hex file with nrf802154_sniffer firmware.
+        :return path to hex file with nrf802154_sniffer firmware
+        """
+
+        return str(os.path.join(get_python_lib(), 'nrf802154_sniffer/nrf802154_sniffer.hex'))
```

Comment:
This won't work cross platform - please use 
```suggestion
        return os.path.normpath(os.path.join(get_python_lib(), 'nrf802154_sniffer/nrf802154_sniffer.hex'))
```

Reply:
I guess it's a matter of taste - maybe with only 3 names to join, we could ditch the `normpath`.

Also: keep in mind that the `str` is redundant.